source/tutorial/hbs-helper.md

```diff
@@ -0,0 +1,81 @@
+So far, our app is directly showing the user data from our Ember Data models.
+As our app grows, we will want to manipulate data further before presenting it to our users.
+For this reason, Ember offers handlebars template helpers to decorate the data in our templates.
+Let's use a handlebars helper to allow our users to quickly see if a property is "standalone" or part of a "Community".
+
+To get started, let's generate a helper for `rental-property-type`, note that helpers must have at least one dash,
```

Comment:
Change the comma before "note" to a period. See https://owl.english.purdue.edu/engagement/2/1/34/.